temporalcli/commands.gen.go

```diff
@@ -0,0 +1,785 @@
+	Parent  *TemporalEnvCommand
+	Command cobra.Command
+}
+
+func NewTemporalEnvGetCommand(cctx *CommandContext, parent *TemporalEnvCommand) *TemporalEnvGetCommand {
+	var s TemporalEnvGetCommand
+	s.Parent = parent
+	s.Command.Use = "get"
+	s.Command.Short = "Print environment properties."
+	if hasHighlighting {
+		s.Command.Long = "\x1b[1mtemporal env get [environment or property]\x1b[0m\n\nPrint all properties of the 'prod' environment:\n\n\x1b[1mtemporal env get prod\x1b[0m\n\ntls-cert-path  /home/my-user/certs/client.cert\ntls-key-path   /home/my-user/certs/client.key\naddress        temporal.example.com:7233\nnamespace      someNamespace\n\nPrint a single property:\n\n\x1b[1mtemporal env get prod.tls-key-path\x1b[0m\n\ntls-key-path  /home/my-user/certs/cluster.key"
+	} else {
+		s.Command.Long = "`temporal env get [environment or property]`\n\nPrint all properties of the 'prod' environment:\n\n`temporal env get prod`\n\ntls-cert-path  /home/my-user/certs/client.cert\ntls-key-path   /home/my-user/certs/client.key\naddress        temporal.example.com:7233\nnamespace      someNamespace\n\nPrint a single property:\n\n`temporal env get prod.tls-key-path`\n\ntls-key-path  /home/my-user/certs/cluster.key"
+	}
+	s.Command.Args = cobra.ExactArgs(1)
+	s.Command.Run = func(c *cobra.Command, args []string) {
+		if err := s.run(cctx, args); err != nil {
+			cctx.Options.Fail(err)
+		}
+	}
+	return &s
+}
+
+type TemporalEnvListCommand struct {
+	Parent  *TemporalEnvCommand
+	Command cobra.Command
+}
+
+func NewTemporalEnvListCommand(cctx *CommandContext, parent *TemporalEnvCommand) *TemporalEnvListCommand {
+	var s TemporalEnvListCommand
+	s.Parent = parent
+	s.Command.Use = "list"
+	s.Command.Short = "Print all environments."
+	s.Command.Long = "List all environments."
+	s.Command.Args = cobra.NoArgs
+	s.Command.Run = func(c *cobra.Command, args []string) {
+		if err := s.run(cctx, args); err != nil {
+			cctx.Options.Fail(err)
+		}
+	}
+	return &s
+}
+
+type TemporalEnvSetCommand struct {
+	Parent  *TemporalEnvCommand
+	Command cobra.Command
+}
+
+func NewTemporalEnvSetCommand(cctx *CommandContext, parent *TemporalEnvCommand) *TemporalEnvSetCommand {
+	var s TemporalEnvSetCommand
+	s.Parent = parent
+	s.Command.Use = "set"
+	s.Command.Short = "Set environment properties."
+	if hasHighlighting {
+		s.Command.Long = "\x1b[1mtemporal env set [environment.property name] [property value]\x1b[0m\n\nProperty names match CLI option names, for example '--address' and '--tls-cert-path':\n\n\x1b[1mtemporal env set prod.address 127.0.0.1:7233\x1b[0m\n\x1b[1mtemporal env set prod.tls-cert-path  /home/my-user/certs/cluster.cert\x1b[0m"
+	} else {
+		s.Command.Long = "`temporal env set [environment.property name] [property value]`\n\nProperty names match CLI option names, for example '--address' and '--tls-cert-path':\n\n`temporal env set prod.address 127.0.0.1:7233`\n`temporal env set prod.tls-cert-path  /home/my-user/certs/cluster.cert`"
```

Comment:
This is because this is just the long description that I took from the old CLI. It is not about args or whatever. It is just English text. Mentioned in the description, but I am trying not to alter docs during this PR, but a new issue is welcome here. As for having cobra properly label positional args, they don't have a good way of specifying this after `[flags]` in their default I don't think.